Move secrets to ENV (#13268)
decidim/decidim: RELEASE_NOTES.md

If not, you need to adapt it to your environment, for instance by changing the decidim docker image to use ruby:3.x.x.

### 1.2. Update your application configuration

> In this version we are changing Decidim's underlaying configuration engine, so, in order to update your application, make sure you read changes about the environment variables (section 3.5 )

Reviewer: I'm proposing a change of the title. Also I like to have copy paste here to ease up the update process

```suggestion
In this version we are changing Decidim's underlaying configuration engine, so, in order to update your application, make sure you read changes about the environment variables (read more about it at "3.4 Deprecation of `Rails.application.secrets`").

```bash
git rm config/secrets.yml
git rm config/initializers/decidim.rb
wget https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb -O config/storage.yml
```
```

You can read more about this change on PR [#14272](https://github.com/decidim/decidim/pull/14272).

### 3.4. Removal of secrets system

If you were already using the Environment Variables for the configuration of your application, then you can remove both the config/secrets.yml and also the decidim initializer:
If you are not using the ENV system, you will need to adjust your application settings to use it.

Before actually removing the initializer, just make sure you do not have any custom configuration.

```bash
git rm config/secrets.yml

> git rm config/initializers/decidim.rb

Reviewer: We should add a command to replacing the config/storage.yml with wget (from [decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb](https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb))

```suggestion
git rm config/initializers/decidim.rb
wget https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb -O config/storage.yml
```

You can read more about this change on PR [#14272](https://github.com/decidim/decidim/pull/14272).

> ### 3.4. Removal of secrets system

Reviewer: This is how it is explained in Rails 7.1 release notes, so let's speak the same language 

```suggestion
### 3.4. Deprecation of `Rails.application.secrets`
```
